sig-auth/charter.md

```diff
@@ -0,0 +1,150 @@
+# SIG Auth Governance
```

Comment:
this jumps into structure/governance, but doesn't actually include the charter (current original charter/goals/non-goals inlined at https://github.com/kubernetes/community/blob/master/sig-auth/README.md#goals)

do we want to move that into this document? separately, those could use some updating/clarification, but probably best to tackle that separately

Reply:
I think the charter / mission statement / goals are more broadly useful (and interesting). What do you think of keeping that content in the README, and instead renaming this document to "governance.md"?

Reply:
sgtm